Makefile

```diff
@@ -393,11 +396,18 @@ proto:
 		--swagger_out=$(PWD)/proto-gen/openapi/ \
 		model/proto/api_v2.proto
 
+	protoc \
+		$(PROTO_INCLUDES) \
+		-I plugin/storage/grpc/proto \
+				--gogo_out=plugins=grpc,$(PROTO_GOGO_MAPPINGS):$(PWD)/plugin/storage/grpc/proto \
```

Comment:
I would prefer folder name to match the package, e.g. `plugin/storage/grpc/proto/storage_v1`